store/src/java/com/zimbra/cs/imap/ImapPath.java

```diff
@@ -240,9 +241,13 @@ String getOwnerAccountId() throws ServiceException {
         } else if (mOwner == null && mCredentials != null) {
             return mCredentials.getAccountId();
         } else if (mOwner == null) {
+            ZimbraLog.imap.debug("mOwner is null");
             return null;
         }
         Account acct = getOwnerAccount();
+        if(acct != null) {
+            ZimbraLog.imap.debug("owner account is %s", acct.getId());
```

Comment:
Worth mentioning the account name?  Fixed a bug once submitted by QA asking for the account name rather than the id.